src/core/control/Control.cpp

```diff
@@ -2403,6 +2406,58 @@ auto Control::openFile(fs::path filepath, int scrollToPage, bool forceOpen) -> b
     return true;
 }
 
+auto Control::changePdfBackground() -> bool {
+
+    LoadHandler loadHandler;
+
+    const fs::path current_filepath = ((*this).doc)->getFilepath();
+    const fs::path current_PDFfilepath = ((*this).doc)->getPdfFilepath();
```

Comment:
According to the [coding conventions](https://github.com/xournalpp/xournalpp/wiki/Coding-conventions#naming-conventions) you should use camelBack for these variables.
```suggestion
    const fs::path currentFilepath = ((*this).doc)->getFilepath();
    const fs::path currentPdfFilepath = ((*this).doc)->getPdfFilepath();
```